salt/modules/boto_iam.py

```diff
@@ -754,21 +780,48 @@ def create_login_profile(user_name, password, region=None, key=None,
 
         salt myminion boto_iam.create_login_profile user_name password
     '''
-    user = get_user(user_name, region, key, keyid, profile)
-    if not user:
-        log.error('IAM user %s does not exist', user_name)
+    conn3 = __utils__['boto3.get_connection']('iam', region=region, key=key,
+                      keyid=keyid, profile=profile)
+    try:
+        info = conn3.create_login_profile(UserName=user_name, Password=password,
+                                          PasswordResetRequired=password_reset_required)
+        log.info('Created profile for IAM user {0}.'.format(user_name))
+        info.pop('ResponseMetadata')
+        return info
+    except botocore.exceptions.ClientError as e:
+        log.debug(e)
+        err = e.response.get('Error', {}).get('Message')
+        log.error('Failed to create profile for IAM user {0}: {1}'.format(user_name, err))
         return False
-    conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+
+
+def update_login_profile(user_name, password=None, password_reset_required=False,
+                         region=None, key=None, keyid=None, profile=None):
+    '''
+    Updates a login profile for the specified user, give the user the
+    ability to access AWS services and the AWS Management Console.
+
+    .. versionadded:: 2015.8.0
```

Comment:
This should be `Fluorine`.

Reply:
Can you also add documentation for each of these kwargs?

Reply:
That wasn't my `versionadded` tag - it's preexisting :)
Hrrm, I thought I DID add docu for my new args... looking....